tests/test_action_checks.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import unittest
+import re
+import sys
+
+class TestActionChecks(unittest.TestCase):
+    def setUp(self):
+        self.review_pattern = r"\b((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted))(?:,((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted)))*\b"
```

Comment:
🔍 Consider adding comments to explain the regex patterns for better maintainability. 📝